srcpkgs/antimicro-qt4/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'antimicro-qt4'
+pkgname=antimicro-qt4
+version=2.23
+revision=1
+wrksrc="antimicro-${version}"
+build_style=cmake
+configure_args="-DUSE_SDL_2=ON -DCMAKE_INSTALL_PREFIX=/usr -DWITH_UINPUT=ON -DWITH_XTEST=ON -DAPPDATA=ON"
+hostmakedepends="pkg-config itstool"
+makedepends="qt5-tools-devel libXtst-devel SDL2-devel qt-devel"
+depends="desktop-file-utils"
+short_desc="GUI program used to map keyboard keys and mouse controls to a gamepad"
```

Comment:
Add a mention to Qt4, to distinguish between the two packages `antimicro` / `antimicro-qt4`

Reply:
A mention? In short_desc? OK